rollup.plugin.sass.mjs

```diff
@@ -72,25 +94,53 @@ export { render };
     'false /** NODE ENVIRONMENT **/'
   );
 
-  code = removeNodeRequire(code, 'chokidar');
-  code = removeNodeRequire(code, 'readline');
+  code = removeCliPkgRequire(code, 'chokidar');
+  code = removeCliPkgRequire(code, 'readline');
+  code = removeNodeRequire(code, 'immutable');
 
   const minified = await minify(code, { module: true });
   code = minified.code;
 
   return code
 }
 
-function removeNodeRequire(code, moduleId) {
-  const requireStr = `require("${moduleId}")`;
+/**
+ * Node modules are required by node_modules/sass/sass.dart.js via `_cli_pkg_requires`.
+ *
+ * This function manually removes unneeded require statements from the source.
+ *
+ * @param {string} code the code to modify
+ * @param {string} moduleId the module identifier found in a require-like statement
+ * @returns {string} the modified code
+ */
+function removeCliPkgRequire(code, moduleId) {
```

Comment:
sort of clarification question - in addition to the mentioned changes w/ adding immutable and the newer API, is this mainly done to reduce bundle size? If so, is it because the way Sass is written doesn't work well w/ tree-shaking?

Reply:
Yes, that and we don't explicitly require it (so it's not in the bundle w/o an additional rollup plugin), nor do I believe we need/use it (IIUC its only used in the new API).

I _would_ like to get a better handle on this before/when we switch to the new Sass API (and I'd like to do it before Dart Sass 2.0). This was just the minimal needed to get Sass updated to a new version, since we hadn't touched it in so long

Reply:
all makes sense I think 👍